### ISSUE TITLE
Podcast Analytics

### ISSUE THREAD

**Issue by ashleydentino** [mamont77/vixi#58]

**Comment by ashleydentino** [comment]
Hey Ruslan - I'm wanting to track my podcast analytics like subscribers and number of downloads. It says I need to get the information from the server that hosts my podcast episodes. Can you please help me configure and view my podcast analytics?

**Comment by mamont77** [comment]
I think we can try this a module https://www.drupal.org/project/download_count.
This is similar to what you want?

**Comment by ashleydentino** [comment]
I'm looking for itunes podcast analytics that will tell me the number of subscribers and the number of downloads. Does this module just count downloads?

Also, I found these notes on the apple website:

Apple does not provide support for third-party podcast analytics, but the following notes may be helpful when implementing analytics on your server or with your web host:

-HTTP 302 responses will be followed to a depth of up to five redirects and will not update the feed URL in the directory.
-The URL preceding GET-style form values (before the first “?”) must end in a media file extension (such as .mp3).

Correct
http://www.podcaster.com/load.mp3?f=&Wipeout.mp3

Incorrect
http://www.podcaster.com/load.php?f=&Wipeout.php

You may need to work with your web host or server administrator to configure the hosting web server appropriately.
------------------
If you think this module is the best fit, we can go ahead and implement. If not, let's keep looking. Thank you.

**Comment by mamont77** [comment]
Thank you. A good found. I will see tomorrow. My sun's birthday today )) 2 years old))

**Comment by ashleydentino** [comment]
Ok, great! Hope your son has a wonderful birthday! Congratulations.

**Comment by mamont77** [comment]
I made a few changes to the site.
Also, I turned on a private method for files. If you see that the site is not working properly, just set here "Public". http://vixi.com/admin/settings/file-system

If it will work, I will implement the code that will do only one directory private.

Now the site can count.
Examples:
http://vixi.com/download_count
http://vixi.com/download_count/1/details

There is bad news also.
In file http://vixi.com/itunes-podcast.xml wrong link
![vixi com itunes-podcast xml - google chrome 04-16-2015 11 34 34](https://cloud.githubusercontent.com/assets/970967/7177129/d98742b0-e42c-11e4-9a47-8ef86a4f64f6.png)
In other words,
The counter works when you click on the link here http://vixi.com/content/vixi-kitchen-ep-1-0 and doesn't work If you press the on the play btn in iTunes.
I'll look for the cause, I think it is a bug in the module, but I'm not sure at this time. I will return to this issue as soon as possible.

**Comment by mamont77** [comment]
Fixed. Ready for testing.

Examples:
http://vixi.com/download_count
http://vixi.com/download_count/1/details

**Comment by ashleydentino** [comment]
Great, thank you! Is this just a link I need to remember? Or is there a way to navigate to it on my site? 

Also can we get rid of the two blocks at the top of the page (screen shot attached)? The banner with the podcast info should only be shown on the homepage. Thank you!
![screen shot 2015-04-20 at 11 43 54 am](https://cloud.githubusercontent.com/assets/7060673/7234065/91ae89e8-e752-11e4-9e38-7dd363d8f62e.png)

**Comment by mamont77** [comment]
>> Is this just a link I need to remember?
I added to http://vixi.com/admin/build/menu-customize/navigation, but for some reason I do not see these links here
![navigation vixi - google chrome 04-20-2015 20 23 46](https://cloud.githubusercontent.com/assets/970967/7236081/3e93529a-e79b-11e4-92d8-41785547249f.png)
Tomorrow I will check the reason.

**Comment by mamont77** [comment]
Link was added.
![odesk team room 04-21-2015 10 57 48](https://cloud.githubusercontent.com/assets/970967/7247738/5ad851ba-e815-11e4-906d-e907c9e78580.png)

Ready for testing.

**Comment by mamont77** [comment]
Sometimes the images are not visible to the visitor. I have to find another way to count the downloads.
